crates/core/src/client/mod.rs

```diff
@@ -315,7 +315,7 @@ impl<P: Provider + Send + Sync> KakarotEthApi<P> for KakarotClient<P> {
 
                     TransactionReceipt {
                         transaction_hash,
-                        transaction_index: None,
+                        transaction_index: Some(U256::ZERO),
```

Comment:
necessary for `forge script` usage, otherwise deserialization error

Reply:
I think that we need to compute the actual transaction index from the filtered block, ie:
- get starknet block
- filter eth tx
- get index of this tx in the filtered block

Reply:
is the index the index of the tx in the whole block or the index of the tx in the subblock?

Reply:
I don't really know when this is used, but I'd say that we filter out the index and return the EVM index because clients may request tx count, then tx by index?